config/initializers/devise.rb

```diff
@@ -245,7 +245,7 @@
   # config.navigational_formats = ['*/*', :html]
 
   # The default HTTP method used to sign out a resource. Default is :delete.
-  config.sign_out_via = :delete
+  config.sign_out_via = :get
```

Comment:
Is this where bad requests get redirected somehow?

Reply:
i don't think this is a good idea. GET for anything that mutates seems bad.

Reply:
i guess so. what's the best way to sign out through a link directly on chrome?

Reply:
ok never mind. there's always simple rest client. I will change it back.